BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -261,14 +290,41 @@ public final class ConvivaAnalytics: NSObject {
         self.isBumper = false
         logger.debugLog(message: "Tracking resumed.")
     }
+
+    /// Attaches a `Player` instance to the Conviva Analytics object.
+    /// We suggest to call this method as soon as the `Player` instance is initialized to not mis any tracking.
```

Comment:
```suggestion
    /// This method should be called as soon as the `Player` instance is initialized to not miss any tracking.
```

Reply:
[b93895e](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/86/commits/b93895e9b63f1d5243c17fc10faf5b6a4ae68cee) + I also added a warning for this use-case: [be311fd](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/86/commits/be311fdffd6697c69bb5cea008f3f00cd170d7d6)